src/main/java/com/matthewglover/tictactoe/core/AlphaBeta.java

```diff
@@ -7,20 +7,21 @@
     private int alpha;
     private int beta;
 
-    public static AlphaBeta run(Game game) {
-        AlphaBeta alphaBeta = new AlphaBeta(game, 0, true, MINIMUM_ALPHA, MAXIMUM_BETA);
+
+    public static AlphaBeta run(Game game, int maxSearchDepth) {
+        AlphaBeta alphaBeta = new AlphaBeta(game, maxSearchDepth, 0, true, MINIMUM_ALPHA, MAXIMUM_BETA);
         alphaBeta.execute();
         return alphaBeta;
     }
 
-    public AlphaBeta(Game game, int depth, boolean isMaximising, int alpha, int beta) {
-        super(game, depth, isMaximising);
+    public AlphaBeta(Game game, int maxSearchDepth, int depth, boolean isMaximising, int alpha, int beta) {
+        super(game, maxSearchDepth, depth, isMaximising);
         this.alpha = alpha;
```

Comment:
Is there a better way to store state than in instance variables? This mutable state bothers me.